introduction.md

```diff
@@ -1 +1,5 @@
-'Hello! I am Gaurav, by profession developer. I am here to learn a lot about git hub. My Favorite code school path is HTML and CSS'
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?